### ISSUE TITLE
Empty state

### ISSUE THREAD

**Issue by vizaiapp** [ksemianov/WrappingHStack#2]
Crashes unwrapping an optional when searching for the max height and the element array is empty

```      
 for index in rows.indices {
            let maxHeightIndex = rows[index].elements
                .max { $0.size.height < $1.size.height }!
                .index
```

**Comment by ksemianov** [comment]
Thank you for the crash report! Could you provide a sample view code that triggers this problem? By having a minimal and complete example, it will be easier to identify the root cause and verify the fix once I have one.

**Comment by ksemianov** [comment]
I've discovered this example that indeed crashes when force unwrapping the optional:

```swift
import SwiftUI
import WrappingHStack

struct ContentView: View {
    var body: some View {
        WrappingHStack {}
    }
}
```

I've applied a fix in version `0.1.3`

Can you please validate that it solves your case as well?

**Comment by ksemianov** [comment]
I'll close the issue for now. Feel free to re-open if `0.1.3` didn't fix it for your case!
